Sources/SWBTestSupport/BuildOperationTester.swift

```diff
@@ -2191,12 +2176,6 @@ extension BuildKey {
     }
 }
 
-extension Registry: @retroactive BuildSystemCache where Key == Path, Value == SystemCacheEntry {
```

Comment:
@compnerd I think if you restore this conformance but drop @retroactive you'll resolve the CI failure